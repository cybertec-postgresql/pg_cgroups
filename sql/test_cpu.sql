-- allow 50% of the availabe CPU
ALTER SYSTEM SET pg_cgroups.cpu_share = 50000;
SELECT pg_reload_conf();
SELECT pg_sleep_for('0.3');
SHOW pg_cgroups.cpu_share;

-- reset
ALTER SYSTEM RESET pg_cgroups.cpu_share;
SELECT pg_reload_conf();
