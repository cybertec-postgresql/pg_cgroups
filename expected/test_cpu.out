-- allow 50% of the availabe CPU
ALTER SYSTEM SET pg_cgroups.cpu_share = 50000;
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

SELECT pg_sleep_for('0.3');
 pg_sleep_for 
--------------
 
(1 row)

SHOW pg_cgroups.cpu_share;
 pg_cgroups.cpu_share 
----------------------
 50000
(1 row)

-- reset
ALTER SYSTEM RESET pg_cgroups.cpu_share;
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

